Sources/ObservableStore/ObservableStore.swift

```diff
@@ -54,6 +54,45 @@ extension ModelProtocol {
     }
 }
 
+extension ModelProtocol {
+    /// Update a child state within a parent state.
+    /// This update offers a convenient way to call child update functions
+    /// from the parent domain, and get parent-domain states and actions
+    /// back from it.
+    ///
+    /// - `get` gets the child's state
+    /// - `set` sets the child's state within the parent state
+    /// - `tag` tags child actions, turning them into parent actions
+    /// - `state` the outer state
+    /// - `action` the inner action
+    /// - `environment` the environment for the update function
+    /// - Returns a new outer state
+    public static func update<ViewModel: ModelProtocol>(
+        get: (Self) -> ViewModel?,
```

Comment:
`get` may return an optional. If `get` returns `nil`, the update is a no-op.

This allows this method to work for both sub-components that always exist, AND for dynamic sub-components (such as list items), which may disappear before one of the actions they generated completed.